indico/modules/users/controllers.py

```diff
@@ -593,12 +593,25 @@ def _send_confirmation(self, email):
     def _process(self):
         form = UserEmailsForm()
         if form.validate_on_submit():
-            self._send_confirmation(form.email.data)
+            email = form.email.data
+
+            # Admin adding email for another user: skip verification
+            if session.user.is_admin and session.user != self.user:
```

Comment:
It would be cleaner if we added an "Admin: skip email validation" checkbox to the form when you're an admin. This would also have the advantage that it's more obvious for admins that the behavior is different.